app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ActiveRecord::Base
+  belongs_to :spot
+
+  validates :rating, presence: true
+  validates :rating, numericality: { greater_than: 0, less_than: 6 }
```

Comment:
Having the validation for numerically will not allow nil values, so the presence validator is somewhat redundant.